pom.xml

```diff
@@ -76,6 +76,18 @@
     <!-- default properties that can be altered on the command line -->
     <fcrepo.test.context.path />
   </properties>
+
+  <distributionManagement>
```

Comment:
I would these changes to live in: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-parent/pom.xml

Reply:
👍